conjureup/controllers/steps/common.py

```diff
@@ -89,4 +89,8 @@ def load_step(step_meta_path):
     # Set environment variables so they can be accessed from the step scripts
     set_env(step_model.additional_input)
 
-    return await utils.run_step(step_model.name, step_model.title, msg_cb)
+    # Set result to be written to
+    app.env['RESULT'] = "conjure-up.{}.{}".format(app.config['spell'],
```

Comment:
Do we want to provide a key prefix value as well, to make it easy for steps to namespace any data they want to store?  Could we handle the prefix automatically via the wrapper?  (If we do that, we could maybe document that the key is just `step_result` or such, without needing an env var for the key at all.)

We could use separate databases, but by default those are limited to 16 and we'd need to separately store the translations from spells to DB index.